extensions/3rdparty/LyricWiki/extras.php

```diff
@@ -81,12 +81,11 @@ function sandboxParse($wikiText)
 ////
 function lw_simpleQuery($queryString){
 	$retVal = "";
-	$db = wfGetDB(DB_SLAVE)->getProperty('mConn');
 
-	// TODO: use Database class instead
-	if($result = mysql_query($queryString,$db)){
-		if(($numRows = mysql_num_rows($result)) && ($numRows > 0)){
-			$row = mysql_fetch_row($result);
+	$dbr = wfGetDB(DB_SLAVE);
+	if($result = $dbr->query( $queryString, __METHOD__ )){
```

Comment:
Does setting `LIMIT 1` for this query make sense (we return the first row only below)?